students/carlos_novoa/lesson02/series.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+"""
+Lesson2, Computing the Fibonacci and Lucas Series
+"""
+
+
+def fibonacci():
+    """Return a list containing a fibonacci series (starting at 0, 1)"""
+    return sum_series(False, 0, 1)
+
+
+def lucas():
+    """Return a list containing a lucas series (starting at 2, 1)"""
+    return sum_series(False, 2, 1)
+
+
+def sum_series(n, a=0, b=1):
+    """
+    Print or return numbers in a fibonacci/lucas series
+
+    :param n: Print option or index of element in series list
+    :param a: First number in series
+    :param b: Second number in series
+    :type n: bool or int
+    :type a: int
+    :type b: int
+    :returns: fibonacci/lucas number(s)
+    :rtype: list
+    """
+
+    if a == 0 and b == 0:
+        return False
+
+    sequence = []
+
+    def series_int(n, a=0, b=1):
+        for i in range(n):
+            a, b = b, a + b
+        return a
+
+    series_range = 10  # limit 10 to avoid maximum recursion error
+    for i in range(series_range):
+        si = series_int(i, a, b)
+        sequence.append(si)
```

Comment:
looks like your fib/lucas sequences can only go up to ten, can you think of another solution that either uses recursion or for/while loop to generate and find seq at any index?